skabase/SKABaseDevice/SKABaseDevice.xmi

```diff
@@ -34,6 +34,22 @@
       <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
       <DefaultPropValue>localhost</DefaultPropValue>
     </deviceProperties>
+    <deviceProperties name="CentralLogger" description="">
+      <type xsi:type="pogoDsl:StringType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+    </deviceProperties>
+    <deviceProperties name="ElementLogger" description="">
+      <type xsi:type="pogoDsl:StringType"/>
+      <status abstract="false" inherited="false" concrete="true" concreteHere="true"/>
+    </deviceProperties>
+    <deviceProperties name="ElementLoglevel" description="">
```

Comment:
This doesn't match up with the design guidelines (SKA1 CONTROL SYSTEM GUIDELINES 000-000000-010 GDL Rev 01A).  The default level should be a property, and then the current value stored as a memorised attribute.
@lorenzopivetta Has that design changed?  It is likely that I have an outdated copy.

From the guideline extract below, it is clear that this XMI file was incomplete.

![image](https://user-images.githubusercontent.com/20420754/50839074-3b10e400-1368-11e9-8816-36da88cb9af4.png)

Reply:
No, the design has not changed. The default values better to be stored into Properties.
As a side note, Rev 01A is the latest draft (currently wip) so your copy is not out of date.